docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1780 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #52.            if approx_P[idx] &lt; discord_dist:
</pre><p>I don't like having <code>continue</code> in the code like this. Why not flip this to:</p><p> </p><pre class="ql-syntax" spellcheck="false">if approx_P[idx] &gt;= discord_dist:
    Q = T[idx:idx+m] 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
I did it first. But, then I used `if ...: continue` to avoid indent  for the rest of the code (as I thought it is easier in terms of readability). 

So, in you mind, is it better to have indent and just avoid `continue`.?

Reply:
<p>Yes. It's subtle but in this case it is simply two nested <code>if</code> statements and not a bunch of <code>if/else</code> and so it's still pretty easy to follow and you can easily see what is going on. In other words, the outer <code>if</code> does not affect the inner <code>if</code> and so my brain is free to forget the outer <code>if</code> once it is processed in this case     </p>